colossalai/kernel/triton/fused_layernorm.py

```diff
@@ -18,7 +18,6 @@ def _layer_norm_fwd_fused(
         X,  # pointer to the input
         Y,  # pointer to the output
         W,  # pointer to the weights
```

Comment:
Wonder if we should revise the name of the kernel, since for now the kernel is no longer supporting bias or fused activation (which is supposed to be supported in a fused kernel).